trendmicro_apex/help.md

```diff
@@ -0,0 +1,134 @@
+# Description
+
+Trend Micro Apex offers modern advanced automated threat detection and response.  Apex agents have more than antivirus
+capabilities, they are an extension of the Apex threat management system.  
+
+# Key Features
+
+* Reporting suspicious ips, urls and other similar content
+* Reporting suspicious files and their contents
+
+# Requirements
+
+* A host running Apex Security Manager
+* API Key
+* Application ID
+
+# Documentation
+
+## Setup
+
+The connection configuration accepts the following parameters:
+
+|Name|Type|Default|Required|Description|Enum|
+|----|----|-------|--------|-----------|----|
+|api_key|credential_secret_key|None|True|API key paired with the Application ID|None|
+|application_id|credential_secret_key|None|True|Application ID to communicate to the Apex Security Manager|None|
+|hostname|string|None|True|URL of the Apex Security Manager|None|
+
+## Technical Details
+
+### Actions
+
+#### Add to UDSO List
+
+This action is used to add an IP, email or similar info to the UDSO list.
+
+##### Input
+
+|Name|Type|Default|Required|Description|Enum|
+|----|----|-------|--------|-----------|----|
+|content|string|None|True|The item to be filed as suspicious|None|
+|data_type|string|URL|True|Format of the data.  URL is max 2047, SHA has is 40 max|['IP', 'URL', 'FILE_SHA1', 'DOMAIN']|
+|expiry_date|int|30|False|Number of days to allow this rule to be active|None|
+|notes|string|None|False|Notes about why the file is being quarantined (256 max)|None|
+|scan_action|string|LOG|True|What action to do with the data sent|['BLOCK', 'LOG']|
+
+Example input:
+
+```
+    "input": {
```

Comment:
All examples inputs and output formatting needs to be updated. Also, only the content of the `input` object.

https://raw.githubusercontent.com/rapid7/insightconnect-plugins/master/dig/help.md